synapse/lib/layer.py

```diff
@@ -3461,6 +3462,7 @@ def _editNodeDel(self, buid, form, edit, sode, meta):
 
             for indx in self.getStorIndx(stortype, valu):
                 self.layrslab.delete(abrv + indx, buid, db=self.byarray)
+                await asyncio.sleep(0)
```

Comment:
Should we wait to pop the valu from the sode and set it dirty until after we've removed these index values?